src/library/SideMenu/Main.tsx

```diff
@@ -160,41 +163,50 @@ export const Main = () => {
       </LogoWrapper>
 
       {pageConfig.categories.map(
-        (category: PageCategory, categoryIndex: number) => (
-          <React.Fragment key={`sidemenu_category_${categoryIndex}`}>
-            {/* display heading if not `default` (used for top links) */}
-            {category.title !== 'default' && (
-              <Heading title={category.title} minimised={sideMenuMinimised} />
-            )}
+        (category: PageCategory, categoryIndex: number) => {
+          return (
+            <React.Fragment key={`sidemenu_category_${categoryIndex}`}>
+              {/* display heading if not `default` (used for top links) */}
+              {category.key !== 'default' && (
+                <Heading
+                  title={tPages(category.key)}
+                  minimised={sideMenuMinimised}
+                />
+              )}
 
-            {/* display category links */}
-            {pagesToDisplay.map((page: PageItem, pageIndex: number) => {
-              return (
-                <React.Fragment key={`sidemenu_page_${pageIndex}`}>
-                  {page.category === category._id && (
-                    <Primary
-                      name={page.title}
-                      to={page.hash}
-                      active={page.hash === pathname}
-                      icon={
-                        page.icon ? (
-                          <FontAwesomeIcon
-                            icon={page.icon}
-                            transform="grow-1"
-                            className="fa-icon"
-                          />
-                        ) : undefined
-                      }
-                      animate={page.animate}
-                      action={page.action}
-                      minimised={sideMenuMinimised}
-                    />
-                  )}
-                </React.Fragment>
-              );
-            })}
-          </React.Fragment>
-        )
+              {/* display category links */}
+              {pagesToDisplay.map((page: PageItem, pageIndex: number) => {
+                return (
+                  <React.Fragment key={`sidemenu_page_${pageIndex}`}>
+                    {page.category === category._id && (
+                      <Primary
+                        name={
+                          i18n.resolvedLanguage === 'en'
```

Comment:
i18n is still being used here. Can you use t instead in the same manner as categories?